CHANGELOG.md

```diff
@@ -3,7 +3,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## [2.2.0] - Unreleased
+## [3.0.0] - 2020-03-11
+## Changed
+- Add support for ruby 2.7
+- Remove support for ruby < 2.3
```

Comment:
What about setting [required_ruby_version](https://guides.rubygems.org/specification-reference/#required_ruby_version) in gemspec file?